.github/workflows/docs.yaml

```diff
@@ -35,5 +38,9 @@ jobs:
 
       - name: build and push
         run: |
-          mike deploy ${{ github.ref_name }} latest --push --update-aliases
+          VERSION="${{ github.ref_name }}"
+          if [[ ${{ github.event_name }} == "workflow_dispatch" ]]; then
+            VERSION="latest"
+          fi
+          mike deploy $VERSION latest --push --update-aliases
```

Comment:
Is this correct?
maybe?

```
mike deploy $VERSION --push --update-aliases
```

as otherwise

```
mike deploy latest latest --push --update-aliases
```

Reply:
ah good catch ! I fixed it in my last commit